%
%
% UCSD Doctoral Dissertation Template
% -----------------------------------
% http:\\ucsd-thesis.googlecode.com
%
%


%% REQUIRED FIELDS -- Replace with the values appropriate to you

% No symbols, formulas, superscripts, or Greek letters are allowed
% in your title.
\title{Statistical Representations Of Microbial Systems}

\author{Yoshiki V\'azquez Baeza}
\degreeyear{2017}

% Master's Degree theses will NOT be formatted properly with this file.
\degreetitle{Doctor of Philosophy} 

\field{Computer Science}
\chair{Rob Knight}
% Uncomment the next line iff you have a Co-Chair
% \cochair{Professor Cochair Semimaster} 
%
% Or, uncomment the next line iff you have two equal Co-Chairs.
%\cochairs{Professor Chair Masterish}{Professor Chair Masterish}

%  The rest of the committee members  must be alphabetized by last name.
\othermembers{
Rachel Dutton\\ 
Pieter Dorrestein\\ 
Jim Hollan\\ 
Larry Smarr\\ 
}
\numberofmembers{4} % |chair| + |cochair| + |othermembers|


%% START THE FRONTMATTER
%
\begin{frontmatter}

%% TITLE PAGES
%
%  This command generates the title, copyright, and signature pages.
%
\makefrontmatter 

%% DEDICATION
%
%  You have three choices here:
%    1. Use the ``dedication'' environment. 
%       Put in the text you want, and everything will be formated for 
%       you. You'll get a perfectly respectable dedication page.
%   
%
%    2. Use the ``mydedication'' environment.  If you don't like the
%       formatting of option 1, use this environment and format things
%       however you wish.
%
%    3. If you don't want a dedication, it's not required.
%
%
\begin{dedication} 
	To my parents, my family and my friends.
\end{dedication}

% You are responsible for formatting here.
%\begin{mydedication} 
%  \vspace{1in}
%  \begin{flushleft}
%    To me.
%  \end{flushleft}
%   
%   \vspace{2in}
%   \begin{center}
%     And you.
%   \end{center}
%
%  \vspace{2in}
%  \begin{flushright}
%    Which equals us.
%  \end{flushright}
%\end{mydedication}



%% EPIGRAPH
%
%  The same choices that applied to the dedication apply here.
%

% The style file will position the text for you.
\begin{epigraph} 
  \emph{The purpose of computing is insight, not numbers}\\
  ---Richard Hamming
\end{epigraph}

%% SETUP THE TABLE OF CONTENTS
%
\tableofcontents
\listoffigures  % Uncomment if you have any figures
\listoftables   % Uncomment if you have any tables

\printglossary[title=List of Abbreviations,toctitle=List of Abbreviations,nonumberlist ]

%% ACKNOWLEDGEMENTS
%
%  While technically optional, you probably have someone to thank.
%  Also, a paragraph acknowledging all coauthors and publishers (if
%  you have any) is required in the acknowledgements page and as the
%  last paragraph of text at the end of each respective chapter. See
%  the OGS Formatting Manual for more information.
%
\begin{acknowledgements} 
 Thanks to whoever deserves credit for Blacks Beach, Porters Pub, and
 every coffee shop in San Diego. 

 Thanks also to hottubs.
\end{acknowledgements}


%% VITA
%
%  A brief vita is required in a doctoral thesis. See the OGS
%  Formatting Manual for more information.
%
\begin{vitapage}
\begin{vita}
  \item[2012] B.~S. in Biomedical Engineering, Universidad Iberoamericana, Ciudad de M\'exico
  \item[2016] M.~Sc. in Computer Science, University of California, San Diego
  \item[2017] Ph.~D. in Computer Science, University of California, San Diego 
\end{vita}


\begin{publications}

    \item \textsl{Author names marked with $\dagger$ indicate shared first co-authorship}.

    \item \textbf{Y. V\'azquez-Baeza}, A. Gonzalez, M. Pirrung, R. Knight. ``Emperor: a tool for visualizing high throughput microbial community data'', \emph{GigaScience}, 2, 2013.

    \item \textbf{Y. V\'azquez-Baeza}, A. Gonzalez, L. Smarr, D. McDonald, J. T. Morton, J. A. Navas-Molina, R. Knight. ``Bringing the Dynamic Microbiome to Life with Animations'', \emph{Cell Host and Microbe}, 11, 2017.

    \item \textbf{Y. V\'azquez-Baeza}, E. R. Hyde, J. S. Suchodolski, R. Knight. ``Dog and human inflammatory bowel disease rely on overlapping yet distinct dysbiosis networks'', \emph{Nature Microbiology}, 1, 2016.

    \item  $\dagger$S. Khanna, \textbf{$\dagger$Y. V\'azquez-Baeza}, A. Gonz\'alez, S. Weiss, B. Schmidt, D. A. Muñiz-Pedrogo, J. F. Rainey 3rd, P. Kammer, H. Nelson, M. Sadowsky, A. Khoruts, S. L. Farrugia, R. Knight, D. S. Pardi, P. C. Kashyap. ``Changes in microbial ecology after fecal microbiota transplantation for recurrent C. difficile infection affected by underlying inflammatory bowel disease'', \emph{Microbiome}. 5, 2017.

    \item J. Halfvarson, C. J. Brislawn, R. Lamendella, \textbf{Y. V\'azquez-Baeza}, W. A. Walters, L. M. Bramer, M. D'Amato, F. Bonfiglio, D. McDonald, A. Gonzalez, E. E. McClure, M. F. Dunklebarger, R. Knight, J. K. Jansson. ``Dynamics of the human gut microbiome in inflammatory bowel disease'', \emph{Nature Microbiology}, 2, 2017.
  
    \item $\dagger$A. Weingarden, A. Gonz\'alez, \textbf{$\dagger$Y. V\'azquez-Baeza}, S. Weiss, G. Humphry, D. Berg-Lyons, D. Knights, T. Unno, A. Bobr, J. Kang, A. Khoruts, R. Knight, M. J. Sadowsky. ``Dynamic changes in short- and long-term bacterial composition following fecal microbiota transplantation for recurrent Clostridium difficile infection''. \emph{Microbiome}. 3, 2015.

    \item \textbf{Y. V\'azquez-Baeza}, C. Callewaert, J. Debelius, E. Hyde, C. Marotz, J. T. Morton, A. Swafford, A. Vrbanac, P. C. Dorrestein and R. Knight. ``Impacts of the human gut microbiome in therapeutics''. \emph{Annual Reviews}. 58, 2017.

    \item \textbf{Y. V\'azquez-Baeza}, A. Gonzalez, Z. Zech Xu, A. Washburne, H. Herfarth, R. B. Sartor, R. Knight. ``Guiding longitudinal sampling in inflammatory bowel diseases cohorts''. \emph{XXXXXX YYYYYYY}. XX, 20XX.

    \item \noindent\rule[0.5ex]{\linewidth}{0.5pt}

    \textsl{The following publications were not included as part of this dissertation, but were also significant byproducts of my doctoral training.}

    \item C. A. Lozupone, J. Stombaugh, A. Gonzalez, G. Ackermann, D. Wendel, \textbf{Y. V\'azquez-Baeza}, J. K. Jansson, J. I. Gordon, R. Knight. ``Meta-analyses of studies of the human microbiota''. \emph{Genome Research}. 23, 2013.

    \item J. A. Navas-Molina, J. M. Peralta-S\'anchez, A. Gonz\'alez, P. J. McMurdie, \textbf{Y. V\'azquez-Baeza}, Z. Xu, L. K Ursell, C. Lauber, H. Zhou, S. Jin Song, J. Huntley, G. L Ackermann, D. Berg-Lyons, S. Holmes,. Gregory Caporaso, R. Knight. ``Advancing our understanding of the human microbiome using QIIME''. \emph{Methods in enzymology}. 531, 2013.

    \item G. E Flores, G. Caporaso, J. B Henley, J. R. Rideout, D. Domogala, J. Chase, J. W Leff, \textbf{Y. V\'azquez-Baeza}, A. Gonzalez, R. Knight, R. R. Dunn, N. Fierer. ``Temporal variability is a personalized feature of the human microbiome''. \emph{Genome biology}. 15, 2014.

    \item S. Lax, D. P Smith, J. Hampton-Marcell, S. M Owens, K. M Handley, N. M Scott, S. M Gibbons, P. Larsen, B. D Shogan, S. Weiss, J. L Metcalf, L. K Ursell, \textbf{Y. V\'azquez-Baeza}, W. Van Treuren, N. A Hasan, M. K Gibson, R. Colwell, G. Dantas, R. Knight, J. A Gilbert. ``Longitudinal analysis of microbial interaction between humans and the indoor environment''. \emph{Science}. 345, 2014.

    \item D. Gevers, S. Kugathasan, L. A Denson, \textbf{Y. V\'azquez-Baeza}, W. Van Treuren, B. Ren, E. Schwager, D. Knights, S. Jin Song, M. Yassour, X. C Morgan, A. D Kostic, C. Luo, A. González, D. McDonald, Y. Haberman, T. Walters, S. Baker, J. Rosh, M. Stephens, M. Heyman, J. Markowitz, R. Baldassano, A. Griffiths, F. Sylvester, D. Mack, S. Kim, W. Crandall, J. Hyams, C. Huttenhower, R. Knight, R. J Xavier. ``The treatment-naive microbiome in new-onset Crohn's disease''. \emph{Cell Host \& Microbe}. 3, 2014.

    \item $\dagger$A. Gonz\'alez, \textbf{$\dagger$Y. V\'azquez-Baeza}, R. Knight. ``SnapShot: the human microbiome''. \emph{Cell}. 158, 2014.

    \item $\dagger$A. Gonzalez, \textbf{$\dagger$Y. V\'azquez-Baeza},  J. B. Pettengill, A. Ottesen, D. McDonald, R. Knight. ``Avoiding Pandemic Fears in the Subway and Conquering the Platypus''. \emph{mSystems}. 1, 2016.

    \item R. A Quinn, J. A Navas-Molina, E. R Hyde, S. Jin Song, \textbf{Y. V\'azquez-Baeza}, G. Humphrey, J. Gaffney, J. J Minich, A. V Melnik, J. Herschend, J. DeReus, A. Durant, R. J Dutton, M. Khosroheidari, C. Green, R. da Silva, P. C Dorrestein, R. Knight ``From sample to Multi-Omics conclusions in under 48 Hours''. \emph{mSystems}. 1, 2016.

    \item J. W Debelius, \textbf{Y. V\'azquez-Baeza}, D. McDonald, Z. Xu, E. Wolfe, R. Knight. ``Turning participatory microbiome research into usable data: lessons from the American Gut Project''. \emph{Journal of microbiology \& biology education}. 1, 2016.

    \item J. Debelius, S. Jin Song, \textbf{Y. V\'azquez-Baeza}, Z. Zech Xu, A. Gonzalez, R. Knight. ``Tiny microbes, enormous impacts: what matters in gut microbiome studies?''. \emph{Genome Biology}. 17, 2017.

    \item J. T Morton, J. Sanders, R. A Quinn, D. McDonald, A. Gonzalez, \textbf{Y. V\'azquez-Baeza}, J. A Navas-Molina, S. Jin Song, J. L Metcalf, E. R Hyde, M. Lladser, P. C Dorrestein, R. Knight. ``Balance trees reveal microbial niche differentiation''. \emph{mSystems}. 2, 2017.

    \item S. Weiss, Z. Zech Xu, S. Peddada, A. Amir, K. Bittinger, A. Gonzalez, C. Lozupone, J. R. Zaneveld, \textbf{Y. V\'azquez-Baeza}, A. Birmingham, E. R Hyde, R. Knight ``Normalization and microbial differential abundance strategies depend upon data characteristics''. \emph{Microbiome}. 5, 2017.

\end{publications}


\end{vitapage}


%% ABSTRACT
%
%  Doctoral dissertation abstracts should not exceed 350 words. 
%   The abstract may continue to a second page if necessary.
%
\begin{abstract}

    Technological developments in the past thirty years, have transformed
    sequencing-based microbiology into a data-intensive field where computing,
    and efficient representations of large datasets are catalyzers of insight
    into omnipresent and complex microbial interactions.

    Notably, classical ecologists have set the foundations for the way we
    analyze these systems, with some techniques dating back to the beginning of
    the twentieth century. In this thesis, we expand and where possible reuse
    these techniques to enhance our understanding of microbial systems.

    To set an appropriate motivation and context to the rest of this work,
    Chapter 1 reviews recent discoveries on the human gut microbiome and how
    the communities within can be determinants on the action of therapeutic
    agents. Next, in Chapter 2, we introduce EMPeror, an interactive
    $\beta$-diversity viewer that has been widely adopted and used to analyze
    microbiome datasets.

    The following three chapters study concrete examples where the microbiome
    has been implicated as a driver or marker for dysbiosis. Chapter 3
    describes how the microbial signature associated with \gls{cd} in humans,
    described in our previous work \cite{RN154}, is overlapping but distinct to
    that of dogs affected with \gls{ibd}. Notably, unlike with humans, dog
    fecal samples alone are strong indicators of the disease. In Chapter 4, we
    study \gls{ibd} from a longitudinal perspective, revealing increased
    volatility in the gut microbiomes of subjects with \gls{ibd}, a property
    that does not appear to be present in unaffected controls. Furthermore, we
    use this as a predicting feature of the disease to improve on the
    classification accuracy possible through a single fecal sample. In Chapter
    5, we study the effect of \glspl{fmt} to treat \gls{cdi} and using the
    techniques described in Chapter 2, we show the first animated visualization
    of this process, a dramatic microbial transformation as the subjects
    recover from all \gls{cdi} symptoms. In addition, for \gls{cdi} patients
    who also suffer from a subtype of \gls{ibd}, a treatment with a \gls{fmt}
    results in an increased number of relapses and decreased microbial
    diversity.

    The closing chapter discusses these results and their possible
    applications, as well as future directions for computationally-centric
    microbiome research.

\end{abstract}


\end{frontmatter}
